include/wx/msw/toplevel.h

```diff
@@ -165,6 +165,9 @@ class WXDLLIMPEXP_CORE wxTopLevelWindowMSW : public wxTopLevelWindowBase
                                           int& x, int& y,
                                           int& w, int& h) const wxOVERRIDE;
 
+    // WM_DPICHANGED handler.
+    bool HandleDPIChange(const wxSize newDPI, const wxRect newRect);
```

Comment:
Sorry, just being curious: why did you decide to not pass the arguments by const reference here?
I am similarly curious about why some member functions names start with `WX`?

Reply:
I think I just forgot to use reference.

Do you mean `WXAdjustToPPI`? I think @vadz added it to emphasize that it is an internal function in a public header.

Reply:
Yes, this should indeed be
```suggestion
    bool HandleDPIChange(const wxSize& newDPI, const wxRect& newRect);
```

Reply:
> Do you mean `WXAdjustToPPI`? I think @vadz added it to emphasize that it is an internal function in a public header.

Yes, it's the convention used for non-public functions that are nevertheless accessible in the headers. This will hopefully prevent people from using them and, at the very least, will prevent accidental conflicts with user-defined methods.